src/server/transaction.cc

```diff
@@ -1597,6 +1473,16 @@ void Transaction::CancelBlocking(std::function<OpStatus(ArgSlice)> status_cb) {
   blocking_barrier_.Close();
 }
 
+bool Transaction::CanRunInlined() const {
```

Comment:
Please add a comment what run inlined means. I know it but better to have a reference in the code.